lib/rubocop/cop/rspec/capybara/visibility_matcher.rb

```diff
@@ -20,33 +20,53 @@ module Capybara
         #   expect(page).to have_selector('.foo', visible: false)
         #
         #   # bad
-        #   expect(page).to have_selector('.foo', visible: true)
+        #   expect(page).to have_css('.foo', visible: true)
         #
         #   # good
-        #   expect(page).to have_selector('.foo', visible: :all)
+        #   expect(page).to have_xpath('.//foo', visible: :all)
```

Comment:
It's better to leave the good examples to be exactly how the bad ones are supposed to be. Otherwise, someone might think that the cop proposes have_xpath instead of have_css